core/server/models/post.js

```diff
@@ -43,8 +43,8 @@ Post = ghostBookshelf.Model.extend({
     defaults: function defaults() {
         let visibility = 'public';
 
-        if (settingsCache.get('labs') && (settingsCache.get('labs').members === true) && settingsCache.get('labs').default_content_visibility) {
-            visibility = settingsCache.get('labs').default_content_visibility;
+        if (settingsCache.get('labs') && (settingsCache.get('labs').members === true) && settingsCache.get('default_content_visibility')) {
+            visibility = settingsCache.get('default_content_visibility');
```

Comment:
This bit can't be behind `labs` because it should not be exposed in https://github.com/TryGhost/Ghost/blob/d899923/core/frontend/services/themes/middleware.js#L47 :see_no_evil: